software/Libraries/onspeed_core/KalmanFilter.cpp

```diff
@@ -1,8 +1,7 @@
 // this library is sourced from https://github.com/har-in-air/Kalmanfilter_altimeter_vario, on Jun 17, 2020
 
-#include "Arduino.h"
```

Comment:
`Arduino.h` was supplying `constrain`; we can replace it with `std::clamp` to drop `Arduino.h` and let this run on the Linux test runner.